src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -1,31 +1,37 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.ComponentModel;
 using System.Runtime.InteropServices;
 using System.Threading;
+using System.Linq;
 using Xunit;
 
 namespace System.Diagnostics.Tests
 {
     public class ProcessThreadTests : ProcessTestBase
     {
-        [ActiveIssue(3202, PlatformID.OSX)]
         [Fact]
         public void TestCommonPriorityAndTimeProperties()
         {
             ProcessThreadCollection threadCollection = _process.Threads;
             Assert.True(threadCollection.Count > 0);
-
             ProcessThread thread = threadCollection[0];
-
-            if (ThreadState.Terminated != thread.ThreadState)
+            try
+            {
+                if (ThreadState.Terminated != thread.ThreadState)
+                {
+                    Assert.True(thread.Id >= 0);
+                    Assert.Equal(_process.BasePriority, thread.BasePriority);
+                    Assert.True(thread.CurrentPriority >= 0);
+                    Assert.True(thread.PrivilegedProcessorTime.TotalSeconds >= 0);
+                    Assert.True(thread.UserProcessorTime.TotalSeconds >= 0);
+                    Assert.True(thread.TotalProcessorTime.TotalSeconds >= 0);
+                }
+            }
+            catch (Win32Exception)
             {
-                Assert.True(thread.Id >= 0);
-                Assert.Equal(_process.BasePriority, thread.BasePriority);
-                Assert.True(thread.CurrentPriority >= 0);
-                Assert.True(thread.PrivilegedProcessorTime.TotalSeconds >= 0);
-                Assert.True(thread.UserProcessorTime.TotalSeconds >= 0);
-                Assert.True(thread.TotalProcessorTime.TotalSeconds >= 0);
+                // Win32Exception is thrown when getting threadinfo fails. 
```

Comment:
Is there any way to tell whether the failure is allowed? Meaning, how do we know after this if it starts always failing due to a bug rather than just sometimes failing due to spurious conditions at run time?